syntacticAnalysis.py

```diff
@@ -81,6 +75,17 @@ def syntacticAnalysis(nlp, line):
         if attr_pos == 0:
             keywords.append((getPhrase(question, attr_pos), "question_word"))
 
+    elif root_pos == 0 or aux_pos == 0:
```

Comment:
Aren't these requirements a bit too loose? Or is that the reason you put them at the end?

Reply:
Yeah, I still need to do more checking on these questions

Reply:
That's not going to be part of the PR?